decode_morse.rb

```diff
@@ -0,0 +1,29 @@
+def decode_word(word)
+  codes = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D',
```

Comment:
- [ ] Good job so far TEAM, Please consider adding the `decode_char` method (e.g. `decode_char(".-")` returns `"A"`.). Currently, we have `decode_massage`, and `decode_word`. Yes, we may use `decode_word` to detect the `char`, but let's separate them to match the [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/morse_code.md#:~:text=Create%20a%20method%20to%20decode%20a%20Morse%20code%20character%2C%20takes%20a%20string%20parameter%2C%20and%20return%20the%20corresponding%20character%20in%20uppercase%20(e.g.%20decode_char(%22.%2D%22)%20returns%20%22A%22).) 👍